recipes/xlwings/meta.yaml

```diff
@@ -0,0 +1,58 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "xlwings" %}
+{% set version = "0.11.2" %}
+{% set sha256 = "f11fb1bf2140a72f4ca8bc054a162c5740a6f93cb7994cac494d05d2cd9723cf" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  entry_points:
+    - xlwings = xlwings.command_line:main
+  skip: True  # [linux]
+
+requirements:
+  build:
+    - python
```

Comment:
It looks like `setuptools` is imported in the `setup.py` of the project. Please include it and add the following line to this recipe:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L26